src/printapp/test/test-api.py

```diff
@@ -141,6 +138,27 @@ def test_print(self):
                 response = app.post('/api/print', data=form_data)
                 self.assertEqual(self._status(response), 201)
 
+            response = app.post('/api/deletejob')
+            self.assertEqual(self._status(response), 404)
+            response = app.post('/api/deletejob/')
+            self.assertEqual(self._status(response), 404)
+            response = app.post('/api/deletejob/invalidJobID')
+            self.assertEqual(self._status(response), 200)
+            
+            response = app.get("/api/uniflowstatus")
+            response_json = flask.json.loads(response.data)
+            self.assertEqual(self._status(response), 200)
+            queue = response_json['queue']
+            self.assertIsNotNone(queue)
+            self.assertNotEqual(queue, [])
+            for document in queue:
+                response = app.post('/api/deletejob/' + document['job_id'])
+                self.assertEqual(self._status(response), 200)
+
+            response = app.get("/api/uniflowstatus")
+            response_json = flask.json.loads(response.data)
+            self.assertEqual(response_json['queue'], [])
+
```

Comment:
You could break this into a different test class and then delete the jobs in the tear down method. Setup and teardown are run before and after every test function in a test class.